src/main/java/org/sagebionetworks/bridge/exporter/handler/HealthDataExportHandler.java

```diff
@@ -82,6 +88,11 @@ protected String getDdbTableKeyValue() {
         return getSynapseTableColumnListCached(schemaFieldDefList);
     }
 
+    @Override
+    public void setBridgeHelper(BridgeHelper bridgeHelper) {
+        this.bridgeHelper = bridgeHelper;
+    }
```

Comment:
To add to this, HealthDataExportHandler already has the BridgeHelper through getManager().getBridgeHelper().

Also, don't override dependency injection setters. This leads to bad things happening.